modules/ROOT/pages/osv2-guide.adoc

```diff
@@ -18,8 +18,7 @@ Use Object Store Connector with Anypoint Studio to construct flows to write to a
 +
 Object Store Connector works with both Object Store version 1 and version 2.
 +
-To use Object Store v2 with Mule 4 apps, do not select an object store global element from the *Object store* menu.
-If you select the object store global element, the app uses the Mule runtime object store, rather than Object Store v2.
+To use Object Store v2 with Mule 4 apps, ensure that you link it to the correct object store configuration reference from the *Object store* menu.
```

Comment:
Is this correct: you link OSv2 to the correct REFERENCE?

Reply:
Yeah, it's a "configuration reference". Maybe it should say "ensure that you select the correct object store configuration reference..."